tests/statick/test_statick.py

```diff
@@ -1203,3 +1206,109 @@ def test_scan_package_with_issues(init_statick_ws):
         shutil.rmtree(os.path.join(os.path.dirname(__file__), "test_package-sei_cert"))
     except OSError as ex:
         print("Error: {}".format(ex))
+
+
+def test_print_no_issues(caplog):
+    """Test that expected error message is logged when no issues are found."""
+    LOGGER.info("Testing now.")
```

Comment:
i see this line in a lot of the new unit tests. it seems like a development debug statement. did you mean to keep it in?

Reply:
It was not needed.